workflow/react-project/src/redux/action-creators/searchCocktailActions.js

```diff
@@ -0,0 +1,39 @@
+import { api } from "../api";
+import { FETCH_SEARCH_COCKTAILS_SUCCESS, FETCH_SEARCH_COCKTAILS_LOADING, FETCH_SEARCH_COCKTAILS_FAIL, CLEAN_SEARCH_RESULTS } from "../types";
+import { toast } from "react-toastify";
+
+export const fetchSearchCocktails = (search) => {
+    return dispatch => {
+        dispatch({
+            type: FETCH_SEARCH_COCKTAILS_LOADING
+        });
+        
+        fetch(`${api.API_URL + api.search + search}`, {
```

Comment:
you can use async/await approach to improve code readability and get rid of all these callbacks